recipes/_omnibus_toolchain.rb

```diff
@@ -17,6 +17,12 @@
 # limitations under the License.
 #
 
+if tagged?('omnibus-toolchain')
```

Comment:
Since only Chef Software, Inc. cares about the angry variant, I think it might be better to expose a well documented toolchain package attribute. Perhaps `node['omnibus']['toolchain_package']`. We would then just set the default value (in `attributes.rb`) to be ''omnibus-toolchain''. We can then add the `tagged?` logic in opscode-ci.